src/Network/Network/help/Get-AzFirewall.md

```diff
@@ -300,7 +300,7 @@ Required: False
 Position: Named
 Default value: None
 Accept pipeline input: True (ByPropertyName)
-Accept wildcard characters: True
+Accept wildcard characters: False
```

Comment:
Why does this change happen?

Reply:
I don't know. It is automatically changed by the script. I did touch any of the md files manually. I can switch them back manually.